lib/MetaCPAN/Web/Controller/Release.pm

```diff
@@ -176,35 +176,42 @@ sub _files_to_categories {
     };
 
     for my $f ( @{$files} ) {
-        if ( length $f->{module} ) {
-            push @{ $ret->{modules} }, $f if $f->{documentation};
-            my %info = (
-                status  => $f->{status},
-                path    => $f->{path},
-                release => $release->{name},
-                author  => $release->{author},
-            );
-            for my $m (
-                is_arrayref( $f->{module} )
-                ? @{ $f->{module} }
-                : $f->{module}
-                )
-            {
-                $info{package} = $m->{name};
-                if ( $f->{documentation} ) {
-                    if (    $m->{name} ne $f->{documentation}
-                        and $m->{indexed}
-                        and $m->{authorized} )
-                    {
-                        push @{ $ret->{provides} },
-                            +{ %info, authorized => 1 };
-                    }
-                }
-                else {
-                    push @{ $ret->{provides} }, \%info;
-                }
+        my %info = (
+            status  => $f->{status},
+            path    => $f->{path},
+            release => $release->{name},
+            author  => $release->{author},
+        );
+
+        my @modules
```

Comment:
As this is the edge case, might be nice to add comment about the situation in which it happens, even add an e.g.?